### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by ncphillips** [ncphillips/gurps-hud#2]
A basic Accessibility Audit should be performed for GURPS HUD, focussing on colour contrast, touch-target sizes, etc.
